apps/sasquatch/src/main/AndroidManifest.xml

```diff
@@ -65,9 +65,6 @@
         <activity
             android:name=".activities.AuthenticationProviderActivity"
             android:label="@string/title_auth" />
-        <activity
```

Comment:
That's a merge of the 2 activity Java files between build variants but the file still exists. You cannot remove the one reference to the manifest.

Reply:
@guperrot Updated in new commit.